controllers/admin/adminmarketingestep3.php

```diff
@@ -366,6 +398,8 @@ private function getHTMLContentDB()
 			if ($pos = strpos($domain_name, ':'))
 				$domain_name = Tools::substr($domain_name, 0, $pos);
 
+			$this->context->smarty->assign('shop_name', Shop::getShops()[Shop::getCurrentShop()]['name']);
```

Comment:
Be careful, this line won't be compatible with PHP 5.2.
You have to store the result of `Shop::getShops()` first, and then use it with `[Shop::getCurrentShop()]['name']`